client/app/views/steps/password.coffee

```diff
@@ -1,14 +1,27 @@
 StepView = require '../step'
+timezones = require '../../lib/timezones'
+
 
 module.exports = class PasswordView extends StepView
+
     template: require '../templates/view_steps_password'
 
     events:
-        'click button': 'onSubmit'
+        'click button': 'doSubmit'
+
+
+    getDataFromDOM: ->
+        # TODO: get user info
+        # from server
+        return {
+            email: 'toto@cozycloud.cc'
+            public_name: 'toto'
+            timezone: timezones[0]
+            password: @$('input[name=password]').val()
+            allow_stats: false
+        }
```

Comment:
Yeah sure @misstick .
1. The previous ```POST /register``` worked with these parameters but now, since the PR #322, it should handle ```password``` parameter only. We can check that together if you still have an issue about that.
2. The ```onboardedSteps``` allows to detect steps already done by the user. This array will be saved in the ```User``` document and the user will redirected by the server according of its value. For each step, the "complete steps" are sent through this array. For the component here, that should be ```['welcome', 'agreement', 'password']```. Don't hesitate if you have other questions about that.